activejob/lib/active_job/test_helper.rb

```diff
@@ -10,6 +10,7 @@ module TestHelper
 
     def before_setup # :nodoc:
       test_adapter = queue_adapter_for_test
+      self.class.to_s.gsub(/Test\z/, "").safe_constantize
```

Comment:
> `ActiveJob::TestCase` does not have `tests` method

People could include `ActiveJob::TestHelper` into an `ActiveSupport::TestCase` class and then it'd be there, iirc.

> I think your concern is correct. I think again about the solution.

❤️